### PR TITLE
Update export.go

### PR DESCRIPTION
**Pull request by faddat**
Make exports faster so that more groups do a mainnet-state based testnet.

### CODE REVIEW

File: export.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ import (
 // exportBufferSize is the number of nodes to buffer in the exporter. It improves throughput by
 // processing multiple nodes per context switch, but take care to avoid excessive memory usage,
 // especially since callers may export several IAVL stores in parallel (e.g. the Cosmos SDK).
-const exportBufferSize = 32
+const exportBufferSize = 64
```
Comment (tac0turtle) [inline export.go:11]: could you provide some benachmarks to compare the difference please

Comment (faddat) [reply export.go:11]: chasing another bug.

Comment (tac0turtle) [reply export.go:11]: gonna throw into draft until we can get a benchmark showing the difference

**Comment by faddat** [comment]
sir, this was a mission without a clear cause....

**Pull request closed without merging**